### PR TITLE
Resolve #596 Add logo and badges to readme

### LINKED ISSUE #596: Add logo and badges to readme

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+[![Build Status](https://travis-ci.com/ventureum/Chainsfr.svg?branch=master)](https://travis-ci.com/ventureum/Chainsfr)
+![License](https://img.shields.io/github/license/ventureum/Chainsfr.svg)
+
+**🌎🚀 Send cryptocurrency by email**
+
+<img src="https://i.imgur.com/yzyOeWC.png" />
+
+- 📚 Read the [FAQ](https://faq.chainsfr.com) first, if you have any questions as a user.
+- 📝 Please report any issues and feedback in the [Chainsfr Chat #General](https://chat.chainsfr.com/channel/general) channel.
```
Comment (InVincible2016) [inline README.md]: Please report any issues and feedback **to** the

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+- 🔧 For technical stuff, use this project's [issues](http://github.com/ventureum/Chainsfr/issues).
+- 🚢 For an overview of what changed with each release, check the [releases](https://github.com/ventureum/Chainsfr/releases)
```
Comment (InVincible2016) [inline README.md]: For an overview of changes in each release

File: README.md (hunk 1 above)
Comment (timothywangdev) [reply README.md]: this is fine.

File: README.md (hunk 2 above)
Comment (timothywangdev) [reply README.md]: fixed.

**Pull request merged**